src/Stratis.StratisD/Program.cs

```diff
@@ -33,8 +30,8 @@ public static void Main(string[] args)
                 // NOTES: running BTC and STRAT side by side is not possible yet as the flags for serialization are static
                 var node = new FullNodeBuilder()
                     .UseNodeSettings(nodeSettings)
-                    .UsePosConsensus()
                     .UseBlockStore()
+                    .UsePosConsensus()
```

Comment:
why the order switch?

Reply:
BlockStore.cs: 114
// Throw if block store was initialized after the consensus.
// This is needed in order to rewind consensus in case it is initialized ahead of the block store.

Reply:
cool